stetho/src/main/java/com/facebook/stetho/inspector/elements/android/AccessibilityNodeInfoWrapper.java

```diff
@@ -0,0 +1,65 @@
+package com.facebook.stetho.inspector.elements.android;
+
+import android.support.v4.view.ViewCompat;
+import android.support.v4.view.accessibility.AccessibilityNodeInfoCompat;
+import android.view.View;
+import android.view.ViewParent;
+
+public class AccessibilityNodeInfoWrapper {
+  private final AccessibilityNodeInfoCompat mNodeInfo;
+  private final View mView;
+  private final ViewParent mParent;
+
+  public AccessibilityNodeInfoWrapper(AccessibilityNodeInfoCompat nodeInfo, View view) {
+    mNodeInfo = nodeInfo;
+    mView = view;
+    mParent = view.getParent();
+  }
+
+  public Boolean getIgnored() {
+    int important = ViewCompat.getImportantForAccessibility(mView);
+
+    if (important == ViewCompat.IMPORTANT_FOR_ACCESSIBILITY_NO ||
+        important == ViewCompat.IMPORTANT_FOR_ACCESSIBILITY_NO_HIDE_DESCENDANTS) {
+      return true;
+    }
+
+    // Go all the way up the tree to make sure no parent has hidden its descendants
+    ViewParent parent = mParent;
+    while (parent instanceof View) {
+      if (ViewCompat.getImportantForAccessibility((View) parent)
+          == ViewCompat.IMPORTANT_FOR_ACCESSIBILITY_NO_HIDE_DESCENDANTS) {
+        return true;
+      }
+      parent = parent.getParent();
+    }
+
+    // AccessibilityNodeInfo is actionable
+    if (mNodeInfo.isClickable() || mNodeInfo.isLongClickable() || mNodeInfo.isFocusable()) {
+      return false;
+    }
+
+    // AccessibilityNodeInfo has text set (commonly set from TextViews and Switches).
+    if (mNodeInfo.getText() != null && mNodeInfo.getText().length() > 0) {
```

Comment:
does `getText()` allocate? if it does, or if we don't know, then we should call it once